src/status_im/chat/views/message/message.cljs

```diff
@@ -100,49 +101,83 @@
   {"\\*[^*]+\\*" {:font-weight :bold}
    "~[^~]+~"     {:font-style :italic}})
 
-(def regx (re-pattern (string/join "|" (map first replacements))))
+(def regx-styled (re-pattern (string/join "|" (map first replacements))))
+
+(def regx-url #"https?://[-a-zA-Z0-9+&@#/%?=~_|!:,.;]*[-a-zA-Z0-9+&@#/%=~_|]")
+
+(defn- parse-str-regx [string regx matched-fn unmatched-fn]
+  (if (string? string)
+    (letfn [(create-unmatched-text [text-seq]
```

Comment:
Any reason to have a `letfn` and not a `defn` ?

Reply:
@jeluard Removed `letfn` now as what it does is just a simple condition check, can be accomplished using `as->` and `if` combined. What do you think?